### DISCUSSION TITLE
Random latency during upscaling phase of inpainting?

### DISCUSSION THREAD

**Discussion by xunkar** [lllyasviel/Fooocus#2413] [Q&A] [answered]
When using inpainting default mode, the process is sometimes very long on the upscaling pre-processing phase. Example log of where it hangs.

[Fooocus] Processing prompts ...
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Image processing ...
Upscaling image with shape (720, 793, 3) ...

The odd part is that the time it takes here is random, it can vary from 5 seconds to 5 minutes. Successive generations using the same parameters and same inpainting mask will result in different waiting times at this step. Is thig a bug or something else on my end?

**Comment by ILikeToasters** [comment, marked as answer]
I run into that sometimes when I am doing other things that use VRAM. Try and minimize your browser windows for a few seconds and see if that gets it started. That is what I need to do sometimes when I run into that.

**Comment by xunkar** [comment]
Sorry for the late reply, but after a lot more time I figured out that this doesn't always work. More precisely, whatever is running on the computer does not always have an impact. Furthermore, it only happens when using full picture inpaint and not improve details mode. Still probably an issue with having only 8 GB VRAM.
